task_2/scss/partners.scss

```diff
@@ -0,0 +1,101 @@
+.partners-section {
```

Comment:
![image](https://user-images.githubusercontent.com/84564899/121712132-121a4d80-cae4-11eb-824f-2bfdadf5340e.png)
content logo is too large and dots are so small

Reply:
I will change that